pattern-demos-product/README.md

```diff
@@ -174,6 +174,16 @@ The validation pattern shows some typical validation scenarios for
   Basic validation:
   ![image](https://github.com/user-attachments/assets/18ce21cc-6714-45c3-b02b-d1859f34c481)
 
+### Zip Demo
+
+This demo show an examples how to use zip feature 
+
+You can upload files from computer, it will process and automatic add to existing zip file or create new zip file if not existed.\
```

Comment:
`You can upload files from your computer. They will be processed and automatically added to an existing zip file. If no ZIP file exists, a new one will be created.\`